### ISSUE TITLE
Datetime picker validation

### ISSUE THREAD

**Issue by matejminar** [BRACKETS-by-TRIAD/admin-generator#39]
I get a following error with datetime picker validation:

<img width="1440" alt="Screenshot 2019-04-09 at 19 02 26" src="https://user-images.githubusercontent.com/9060071/55819974-4e178980-5afa-11e9-88a6-173593e09d87.png">

This issue is related to this change:
https://github.com/date-fns/date-fns/blob/master/docs/unicodeTokens.md

It could be resolved in `src/Generate/Traits/Columns.php`, line `108` and `113`.

Basically we need to change `YYYY-MM-DD HH:mm:ss` to `yyyy-MM-dd HH:mm:ss`

**Comment by dejwCake** [comment]
Fixed in release v3.0.2.
